pyproject.toml

```diff
@@ -19,6 +19,9 @@ dataclasses = { version = "^0.7", python = ">=3.6, <3.7" }
 grpclib = "^0.3.1"
```

Comment:
After rebasing I'd suggest adding a "docs" poe task to build sphinx locally

Reply:
Also, it's easily to make syntax errors with sphinx, and since sphinx builds often aren't exactly instant, I'd suggesting adding both a poe task and a CI step that runs rstcheck (which will need to be added as a dev dep)

Reply:
So what I was thinking was to setup a test that just runs https://pypi.org/project/restructuredtext-lint and checks if that finds any errors